pages.ja/linux/ip.md

```diff
@@ -28,7 +28,7 @@
 
 `sudo ip {{[l|link]}} {{[s|set]}} {{インターフェース}} {{up|down}}`
 
-- インターフェースにIPアドレスを追加/削除する:
+- インターフェースに IP アドレスを追加/削除する:
```

Comment:
I do not know enough about japanese to know if this is right. Otherwise looks good.

Reply:
I mean I didn't change anything I believe it was Prettier just adding whitespace.

<img width="688" height="522" alt="image" src="https://github.com/user-attachments/assets/deb142ff-74a3-400d-9b72-b551a9e2a1ce" />